### PR TITLE
Chore/migration npm deps to forge

### PR DESCRIPTION
**Pull request by parth-15**

### CODE REVIEW

**Review by PrinceRuler — APPROVED** [review]

File: package.json [hunk 1]
```diff
@@ -69,17 +75,9 @@
...
-    "@aave/core-v3": "^1.17.2",
-    "@aave/deploy-v3": "^1.55.3",
-    "@aave/periphery-v3": "^2.0.1",
-    "@aave/safety-module": "^1.0.3",
-    "@openzeppelin/contracts": "^4.5.0"
-  },
   "overrides": {
     "@nomicfoundation/hardhat-toolbox": {
-      "@nomiclabs/hardhat-ethers": "npm:hardhat-deploy-ethers"
+      "@nomiclabs/hardhat-ethers": "npm:hardhat-deploy-ethers@0.3.0-beta.13"
```
Comment (miguelmtzinf) [inline package.json:80]: why this change is necessary? would prefer to avoid "beta" packages

**Review by miguelmtzinf — APPROVED** [review]

File: package.json (hunk 1 above)
Comment (miguelmtzinf) [reply package.json:80]: Let's keep it as it is, given that there are [compatibility issues](https://github.com/wighawag/hardhat-deploy-ethers/issues/27#issuecomment-1431411121) between hardhat-deploy and hardhat-toolbox..,
